src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+
+    public String getColor() {
+        return color;
+    }
+    public void setColor(String color) {
+        this.color = color;
+    }
+    public Ball(String color, int number) {
```

Comment:
Please pay attention to access modifiers for constructors as well. If the intention is for the constructor to be used within the package or by subclasses, you can leave it as package-private, otherwise, you might want to explicitly declare it as public.